components/ArticlesList.jsx

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ArticleCard } from "./ArticleCard";
+import { Container, Grid } from "@mui/material";
+
+export const ArticlesList = () => {
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get("https://bw-news-app.onrender.com/api/articles")
```

Comment:
It would be great if we could extract this api call out into a separate api.js file in root, this will keep our components tidy going forward and allow these api calls to be reused if we wanted to :)